### ISSUE TITLE
BUG: CSG + calibration step

### ISSUE THREAD

**Issue by remi-braun** [sertit/eoreader#48]
For some reason, applying a calibration step in SNAP to some CSG products outputs an empty raster.
The workaround is to use the `cplx_no_calib_preprocess_default.xml` graph instead.

```python
import os
from eoreader.reader import Reader
from eoreader.bands import VV
from eoreader.env_vars import PP_GRAPH

os.environ[PP_GRAPH]="path/to/eoreader/data/cplx_no_calib_preprocess_default.xml"

path = r"CSG_SCS_1254349-1052847"
prod = Reader().open(path)
prod.load(VV)
```

This may be worth investiguating (when having more CSG data?)

**Comment by remi-braun** [comment]
SNAP 9.0 doesn't seem to fix this. Nothing found on STEP forum either...

**Comment by remi-braun** [comment]
By default, calibration is not applied to slant range `CSG` data, avoiding producing an empty raster. Implemented in commit 1ef71ff315dc6aca540d4c1796528b6f8e2cda8b
